.travis.yml

```diff
@@ -14,7 +14,7 @@ before_install:
 env:
   - COMMAND=eslint
   - COMMAND=stylelint
-  - COMMAND=test
+  - COMMAND=unit-test
```

Comment:
Where does the coverage report get submitted to Coveralls? That should be a command inside `.travis.yml`. Also take into account that there are several jobs in the text matrix, see:
https://github.com/weldr/bdcs-api-rs/blob/master/.travis.yml#L26